test/e2e/framework/util.go

```diff
@@ -208,6 +208,9 @@ var (
 	// BusyBoxImage is the image URI of BusyBox.
 	BusyBoxImage = imageutils.GetE2EImage(imageutils.BusyBox)
 
+	// AgnHostImage is the image URI of AgnHost
+	AgnHostImage = imageutils.GetE2EImage(imageutils.Agnhost)
```

Comment:
Why do we need a var for this? it's only used once

Reply:
It was used in multiple places. I replaced them all with this var